extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/cors/CORSHandlerTestWildcardOriginCase.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.vertx.http.cors;
+
+import static io.restassured.RestAssured.given;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class CORSHandlerTestWildcardOriginCase {
```

Comment:
Hi @mcserra, interesting case, I actually think it should be `true` here because a precise match possible, I think we may need to tweak the check, please see below

Reply:
Yes, sure, I suppose if the origin match the '*' is irrelevant